aws-rds-certificates.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: aws-rds-certificates
+  version: "20240712"
+  epoch: 0
+  description: AWS RDS CA certificates
+  copyright:
+    - license: Apache-2.0
```

Comment:
Is this the correct license? I couldn't find any documentation about the license of these certs.

Reply:
it's not. I dont know what it is so i just left it like that. will update before merging